packages/server/tests/acceptance/conformityTests.spec.ts

```diff
@@ -5,6 +5,8 @@ const directoryPath = path.resolve(__dirname, '../../../../node_modules/executio
 const axios = require('axios');
 const openRpcData = require('../../../../docs/openrpc.json');
 require('dotenv').config();
+import Ajv from 'ajv';
+import addFormats from 'ajv-formats';
 import { signTransaction } from '../../../relay/tests/helpers';
 import { expect } from 'chai';
 let currentBlockHash;
```

Comment:
Please clean up this session. I noticed that both `import` and `require` syntaxes are being used for importing dependencies. To conform to ES6 standards, please use `import` instead of `require`. Additionally, let's improve readability by adding some empty lines between the imports section and the variable declaration section.